core/controllers/classifier.py

```diff
@@ -79,7 +79,10 @@ def verify_signature(message, vm_id, received_signature):
     if secret is None:
         return False
 
-    generated_signature = generate_signature(secret, message)
+    if message is None:
```

Comment:
@seanlip. My guess is that, when I implemented this function, it worked only for the StoreJobResult controller. This way, if no message is passed, it means it is being called by the NextJob Controller.